doc/developers.rst

```diff
@@ -66,37 +68,82 @@ Install Autosklearn
       
       .. code-block:: console
       
-         $ brew install swig # (or apt-get)
+         (venv) $ brew install swig # (or apt-get)
    
    2. Install gcc (MacOS only)
-   Use your package manager to install gcc (necessary for xgboost)
-   
-      .. code-block:: console
       
-         $ brew install gcc # (or apt-get)
-   
-   3. Install xgboost and autosklearn
+      Use your package manager to install gcc (necessary for xgboost)
    
       .. code-block:: console
       
-         $ export CC=/path/to/installed/gcc; export CXX=/path/to/installed/g++; pip install xgboost
```

Comment:
Exports shouldn't be necessary here, this will set CC and CXX for the parent shell which may have unintended consequences to any development tools in the same session.

Reply:
And.... that was a deleted line. Nevermind. However, should we have some variety of this line? Unless we are trusting the new system to install xgboost. Even in that case I imagine the environmental variables would need to be set.

Reply:
I am suggesting that it is a user's problem to install xgboost and autosklearn since we have left those as optional dependencies.